examples/plugins/example/example.cpp

```diff
@@ -43,23 +43,24 @@ example_actor::behavior_type
 example(example_actor::stateful_pointer<example_actor_state> self) {
   return {
     [=](atom::config, record config) {
-      VAST_TRACE(self, "sets configuration", config);
+      VAST_TRACE("{} sets configuration {}", detail::id_or_name(self), config);
```

Comment:
Do we always need to pass `id_or_name`, or can we make some use of SFINAE to apply `id_or_name` to every passed actor handle?

Reply:
I think we will ultimately solve this by having formatters that can handle the various kinds of actor handles directly, but I would propose to do this as a separate refactoring.

Reply:
Yes. Ideally we'd get rid of the need to use the `caf::stringification_inspector` (used by `caf::deep_to_string`) as much as possible.